### ISSUE TITLE
experiments on DanceTrack

### ISSUE THREAD

**Issue by Kroery** [vukasin-stanojevic/BoostTrack#5]
Have you conducted any experiments on DanceTrack? If so, how should hyperparameters, such as lambda_iou, lambda_mhd, and lambda_shape, be set for DanceTrack?

**Comment by vukasin-stanojevic** [comment]
Hello, we did not experiment with DanceTrack, but I would try with lower lambda_iou and higher lambda_shape.

**Comment by vukasin-stanojevic** [comment]
And try play with softmax temperature when using Mahalanobis distance similarity. Hope this helps. 
@Kroery
